full-stack-tests/backend/src/integration/Checkpoints.test.ts

```diff
@@ -32,14 +32,14 @@ describe.skip("Checkpoints", () => {
   let testITwinId2: GuidString;
   let testChangeSet2: ChangesetProps;
 
-  const blockcacheDir = path.join(KnownTestLocations.outputDir, "blockcachevfs");
+  const cloudcacheDir = path.join(KnownTestLocations.outputDir, "blockcachevfs");
```

Comment:
should probably change the name of the directory too.

Reply:
changed